### DISCUSSION TITLE
kpanda/user-guide/storage/sc-share/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1719] [Announcements]
# kpanda/user-guide/storage/sc-share/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/kpanda/user-guide/storage/sc-share/

**Comment by liyuerich** [comment]
有时候创建出来的sc是有所有命名空间授权， 有时候需要授权。 这个能说明一下吗？
